manifests/p/Parsec/Parsec/150.75/Parsec.Parsec.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Parsec.Parsec
+PackageVersion: "150.75"
```

```suggestion
PackageVersion: "150-82a"
```
This must match Control Panel to prevent upgrade loops while running `winget upgrade`.

---

You are correct. I have applied the suggestions. Thanks for pointing this out!